catalog/dags/providers/provider_api_scripts/europeana.py

```diff
@@ -223,7 +244,61 @@ def get_batch_data(self, response_json: dict) -> None | list[dict]:
         return response_json.get("items")
 
     def get_record_data(self, data: dict) -> dict:
-        return self.record_builder.get_record_data(data)
+        return self.record_builder.get_record_data(
+            data | {"item webresource": self._get_additional_item_data(data)}
```

Comment:
Could we change the key to just `webresource` (or `item_webresource` with the underscore)?